recipe/meta.yaml

```diff
@@ -17,6 +17,8 @@ requirements:
   build:
     - {{ compiler('cxx') }}
     - cmake
+    - make  # [unix]
+    - {{ compiler('cxx') }}
```

Comment:
That second `{{ compiler('cxx') }}` is redundant with the one a few lines above...

Reply:
good catch!